tests/misc/test_readline.py

```diff
@@ -21,144 +21,256 @@
-        mocked_qapp.focusWidget = mock.Mock(return_value=self.qle)
-        self.bridge.rl_delete_char()
-        self.qle.del_.assert_called_with()
-
-    def test_rl_backward_delete_char(self, mocked_qapp):
-        """Test rl_backward_delete_char."""
-        mocked_qapp.focusWidget = mock.Mock(return_value=self.qle)
-        self.bridge.rl_backward_delete_char()
-        self.qle.backspace.assert_called_with()
-
-    def test_rl_unix_line_discard(self, mocked_qapp):
-        """Set a selected text, delete it, see if it comes back with yank."""
-        mocked_qapp.focusWidget = mock.Mock(return_value=self.qle)
-        self._set_selected_text("delete test")
-        self.bridge.rl_unix_line_discard()
-        self.qle.home.assert_called_with(True)
-        assert self.bridge._deleted[self.qle] == "delete test"
-        self.qle.del_.assert_called_with()
-        self.bridge.rl_yank()
-        self.qle.insert.assert_called_with("delete test")
-
-    def test_rl_kill_line(self, mocked_qapp):
-        """Set a selected text, delete it, see if it comes back with yank."""
-        mocked_qapp.focusWidget = mock.Mock(return_value=self.qle)
-        self._set_selected_text("delete test")
-        self.bridge.rl_kill_line()
-        self.qle.end.assert_called_with(True)
-        assert self.bridge._deleted[self.qle] == "delete test"
-        self.qle.del_.assert_called_with()
-        self.bridge.rl_yank()
-        self.qle.insert.assert_called_with("delete test")
-
-    def test_rl_unix_word_rubout(self, mocked_qapp):
-        """Set a selected text, delete it, see if it comes back with yank."""
-        mocked_qapp.focusWidget = mock.Mock(return_value=self.qle)
-        self._set_selected_text("delete test")
-        self.bridge.rl_unix_word_rubout()
-        self.qle.cursorWordBackward.assert_called_with(True)
-        assert self.bridge._deleted[self.qle] == "delete test"
-        self.qle.del_.assert_called_with()
-        self.bridge.rl_yank()
-        self.qle.insert.assert_called_with("delete test")
-
-    def test_rl_kill_word(self, mocked_qapp):
-        """Set a selected text, delete it, see if it comes back with yank."""
-        mocked_qapp.focusWidget = mock.Mock(return_value=self.qle)
-        self._set_selected_text("delete test")
-        self.bridge.rl_kill_word()
-        self.qle.cursorWordForward.assert_called_with(True)
-        assert self.bridge._deleted[self.qle] == "delete test"
-        self.qle.del_.assert_called_with()
-        self.bridge.rl_yank()
-        self.qle.insert.assert_called_with("delete test")
-
-    def test_rl_yank_no_text(self, mocked_qapp):
-        """Test yank without having deleted anything."""
-        mocked_qapp.focusWidget = mock.Mock(return_value=self.qle)
-        self.bridge.rl_yank()
-        assert not self.qle.insert.called
+def bridge():
+    """Fixture providing a ReadlineBridge."""
+    return readline.ReadlineBridge()
+
+
+def test_none(bridge, qtbot):
```

Comment:
I'm assuming you are now also a fan of writing test functions instead of test methods, am I correct? :smile: